--- build-aux/config.sub.orig	2011-04-27 10:28:01.000000000 +0200
+++ build-aux/config.sub	2011-04-27 10:32:27.000000000 +0200
@@ -4,7 +4,7 @@
 #   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009
 #   Free Software Foundation, Inc.
 
-timestamp='2009-11-20'
+timestamp='2011-04-27'
 
 # This file is (in principle) common to ALL GNU software.
 # The presence of a machine in this file suggests that SOME GNU software
@@ -280,6 +280,8 @@
 	| mn10200 | mn10300 \
 	| moxie \
 	| mt \
+        | mtalpha \
+        | mtsparc \
 	| msp430 \
 	| nios | nios2 \
 	| ns16k | ns32k \
@@ -366,6 +368,8 @@
 	| mipstx39-* | mipstx39el-* \
 	| mmix-* \
 	| mt-* \
+        | mtalpha-* \
+        | mtsparc-* \
 	| msp430-* \
 	| nios-* | nios2-* \
 	| none-* | np1-* | ns16k-* | ns32k-* \
