--- build-aux/config.sub.orig	2013-09-16 17:06:11.000000000 +0200
+++ build-aux/config.sub	2013-09-16 17:08:11.000000000 +0200
@@ -291,6 +291,8 @@
 	| mn10200 | mn10300 \
 	| moxie \
 	| mt \
+	| mtalpha \
+	| mtsparc \
 	| msp430 \
 	| nds32 | nds32le | nds32be \
 	| nios | nios2 | nios2eb | nios2el \
@@ -408,6 +410,8 @@
 	| mipstx39-* | mipstx39el-* \
 	| mmix-* \
 	| mt-* \
+	| mtalpha-* \
+	| mtsparc-* \
 	| msp430-* \
 	| nds32-* | nds32le-* | nds32be-* \
 	| nios-* | nios2-* | nios2eb-* | nios2el-* \
