--- build-aux/config.sub.orig	2011-04-27 10:28:01.000000000 +0200
+++ build-aux/config.sub	2011-04-27 10:32:27.000000000 +0200
@@ -280,6 +280,8 @@
 	| mn10200 | mn10300 \
 	| moxie \
 	| mt \
+        | mtalpha \
+        | mtsparc \
 	| msp430 \
 	| nios | nios2 \
 	| ns16k | ns32k \
@@ -366,6 +368,8 @@
 	| mipstx39-* | mipstx39el-* \
 	| mmix-* \
 	| mt-* \
+        | mtalpha-* \
+        | mtsparc-* \
 	| msp430-* \
 	| nios-* | nios2-* \
 	| none-* | np1-* | ns16k-* | ns32k-* \
